src/components/itemsComponents/ProductCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { css } from "@/styled-system/css";
+import { Product } from "@/types/products.type";
+import favoriteIcon from "@/assets/icons/heart_ic.svg";
+import Image from "next/image";
+import React from "react";
+import { hstack } from "@/styled-system/patterns/hstack.mjs";
+// index를 프롭으로 받게되면서 범용성이 많이 떨어짐 전체 상품에서도 쓸만했는데..
+function ProductCard({ item, index }: { item: Product; index?: number }) {
```

Comment:
해당 컴포넌트의 props를 위에 따로 타입으로 빼줘도 좋겠네요!
```suggestion
interface ProductCardProps {
item: Product; 
index?: number
}

function ProductCard({ item, index }: ProductCardProps) {
```